configure/RULES_TOP

```diff
@@ -61,6 +61,24 @@ ifndef DISABLE_TOP_RULES
   runtests test-results:
 	@$(SHOWTESTFAILURES)
 
+  coverage:
+	@mkdir -p coverage
+	@if ! find . -name '*.gcda' | grep -q .; then \
+	  echo "No .gcda files found! Did you build with coverage flags?"; \
```

Comment:
Either here or in the help text it might be good to specify _what_ the coverage flags are.